src/tools/migrations/steps/steps/template.ts

```diff
@@ -1,5 +1,5 @@
 import { BaseProtocol } from 'server/db/db'
 
-export default async (client: BaseProtocol): Promise<void> => {
+export default async (_client: BaseProtocol): Promise<void> => {
```

Comment:
this shouldnt be renamed

Reply:
update `.eslintignore` is better like it was

Reply:
@sorja I had to exclude it from both tsconfig and eslintignore https://github.com/openforis/fra-platform/pull/5305/commits/d04d9b26db5f9eb0463576262ffd8ae84759cf20